types/postcss/extensions.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as postcss from 'postcss';
+declare module 'postcss' {
+
+	interface NodeSource {
+		lang?: string;
+
+	}
```

Comment:
[note] This `lang` property seems an extension by `postcss-syntax`. I cannot find it in the PostCSS core repository.

https://github.com/gucong3000/postcss-syntax/blob/0bca5a408f12891a622fd6149d8f23d3bd98df86/syntax.js#L17

Reply:
Do you know what the best way of handling this is?

Reply:
No, I'm not sure now. 😣 
This solution seems best for us now.